webpack/scenes/ContentViews/Table/tableDataGenerator.js

```diff
@@ -25,13 +25,13 @@ const buildRow = (contentView) => {
   const { last_sync_words: lastSyncWords } = lastTask || {};
   const row = [
     { title: <ContentViewIcon composite={composite ? true : undefined} /> },
-    { title: <Link to={urlBuilder('labs/content_views', '', id)}>{name}</Link> },
+    { title: <Link to={`${urlBuilder('labs/content_views', '')}${id}`}>{name}</Link> },
```

Comment:
Was this broken?

Reply:
Kind of, I wanted the pattern to look like 
content_views/6#/versions/11/dockerTags
not 
content_views/6/#/versions/11/dockerTags

Reply:
the trailing '/' was why this was changed.

Reply:
both would work but to make it consistent with the hosts page I wanted to eliminate the trailing /.